draft-ietf-opsawg-pcapng.md

```diff
@@ -2220,6 +2228,182 @@ The following is a list of Secrets Types.
+For the global form, X corresponds to latitude, Y to longitude, and Z to altitude.
+Yaw is about the X/Y or Lat/Lng plane with zero facing along Y or North,
+pitch is applied with this yaw offset, and roll is applied to the combination of both.
+
+Error options provide an optional mechanism for attaching error information to measurements,
+and are only valid where the corresponding measurement (location, velocity, or orientation)
+option is specified. Should an error option be present without the corresponding measurement,
+this option should be ignored.
+
+All the information fields are defined as options in order to support
+systems that do not have a complete set of information. Therefore,
+In addition to the options defined in {{section_opt}}, the following
+options are valid within this block:
+
+| Name | Code | Length | Multiple allowed? |
+| local_location | 2 | 12 | no |
+| local_location_error | 3 | 12 | no |
+| global_location | 4 | 12 | no |
+| global_location_error | 5 | 12 | no |
+| orientation | 6 | 12 | no |
+| orientation_error | 7 | 12 | no |
+| velocity | 8 | 12 | no |
+| velocity_error | 9 | 12 | no |
+{: #options_cib title='Capture Information Block Options'}
+
+
+{: indent='8'}
+cib_local_location:
+: The cib_local_location option specifies the local location of the packet
+  capture or interface in an abitrary plane; location is stored as three
+  32-bit floats representing X, Y (in plane), and Z (altitude) offsets in
+  metres.
+
+  Example: '41 24 cc cd 41 73 33 33 3f cc cc cd' decodes to
+  x: 10.3m, y: 15.2m, z: 1.6m
+{: vspace='0'}
+
+{: indent='8'}
+cib_local_location_error:
+: The cib_local_location_error option specifies the error in local location
+  capture;  this is stored as three 32-bit floats in the same order and
+  representation as cib_local_location.
+
+  Example: '3f 80 00 00 3f 99 99 9a 3f cc cc cd' decodes to
+  x_error: 1.0m, y_error: 1.2m, z_error: 1.6m
+{: vspace='0'}
+
+{: indent='8'}
+cib_global_location:
+: The cib_global_location option specifies the global location of the
+  packet capture or interface; location is stored as three 32-bit floats
+  representing latitude and longitude in degrees, and altitude in
+  metres.
+
+  Example: 'c2 13 64 ce 43 2e c2 90 41 88 00 00' decodes to
+  lat: -36.8484437° lng: 174.7600023° alt: 17.0m
+{: vspace='0'}
+
+{: indent='8'}
+cib_global_location_error:
+: The cib_global_location_error option specifies the error in global location
+  capture; this is stored as three 32-bit floats in the same order and
+  representation as cib_global_location.
+
+  Example: '3f 80 00 00 3f 99 99 9a 3f cc cc cd' decodes to
+  lat_error: 0.1° lng_error: 0.2° alt_error: 1.6m
+{: vspace='0'}
+
+{: indent='8'}
+cib_orientation:
+: The cib_orientation option specifies the orientation of the packet
+  capturing antenna; orientation is stored as three 32-bit floats
+  representing yaw, pitch, and roll in radians.
+  This is relative to the zero frame for global or local location.
```

Comment:
The zero frame for global location is presumably the frame whose origin is at the intersection of the [IERS Reference Meridian](https://en.wikipedia.org/wiki/IERS_Reference_Meridian) and the Equator, at an altitude that's relative to some reference point at that location ([mean sea level](https://en.wikipedia.org/wiki/Sea_level), which would make it [height above mean sea level](https://en.wikipedia.org/wiki/Height_above_mean_sea_level)?).

The plane for local location is presumably a plane tangent to the Earth's surface at some point; is that a fixed point, or could it move relative to the origin of the zero frame for global location?